docs/scripts_md/imaging_upload_file.md

```diff
@@ -0,0 +1,93 @@
+# NAME
+
+imaging\_upload\_file.pl -- a single step script for the imaging pre-processing
+and insertion pipeline sequence
+
+# SYNOPSIS
+
+perl imaging\_upload\_file.pl &lt;/path/to/UploadedFile> \`\[options\]\`
+
+Available options are:
+
+\-profile      : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-upload\_id    : The upload ID of the given scan uploaded
```

Comment:
Very picky comment: to be consistent with the two other options, the letter T in "The upload ID" should be lower case